.config/webpack/base.js

```diff
@@ -1,12 +1,15 @@
-
 const path = require('path');
+const WebpackDashDynamicImport = require('webpack-dash-dynamic-import');
```

Comment:
The plugin for Dash  chunk path resolution https://github.com/plotly/webpack-dash-dynamic-import